index.html

```diff
@@ -160,20 +160,21 @@ <h3 id="h3-lang">
          <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         </div>
         <div>
-         <form class="contact" action="https://formspree.io/f/myylobqj" method="post">
+         <form class="contact" action="https://formspree.io/f/mqkoyzva" method="post">
            <input type="text" name="name" placeholder="Full name" maxlength="30" required>
            <input type="email" name="email" placeholder="Email address" required>
-           <textarea name="message" rows="3" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
-         </form>
+           <textarea name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+           <button type="submit" name="submit" id="contact-button">Get in touch</button>
```

Comment:
- [ ] Nice work on the button, it looks exactly like the Figma except for a tiny detail that was missed which is the button interactions 🔽 please implement these interactions they are essential and improve accessibility

![2023-03-28_230859](https://user-images.githubusercontent.com/104319462/228387123-cbfb79be-318f-41e4-9f0b-cff2fb3b454f.png)

- [ ] [OPTIONAL] Currently when I hover over the button my cursor icon doesn't switch to the hand icon, please put a `cursor: pointer` property to your button, its a common practice in interfaces developments, and users will always expect this behaviour when they hover over buttons.